var/spack/repos/builtin/packages/acts-dfelibs/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class ActsDfelibs(CMakePackage):
+    """Acts fork of dfelibs, a header-only untility library for C++."""
+
+    homepage = "https://github.com/acts-project/dfelibs"
+    url = "https://github.com/acts-project/dfelibs/archive/refs/tags/v20211029.tar.gz"
+    list_url = "https://github.com/acts-project/dfelibs/tags"
```

Comment:
I'm curious: is this list_url necessary (again)?

Spack should be able to deduce the list URL for GitHub tag archives automatically, but occasionally GitHub changes something (including hiding html content behind dynamically loading JavaScript calls...) that breaks what we do.

Reply:
I guess not; I will remove it and test it.